compiler/analysis/info.go

```diff
@@ -97,158 +122,206 @@ func AnalyzePkg(files []*ast.File, fileSet *token.FileSet, typesInfo *types.Info
 	return info
 }
 
-func (c *FuncInfo) Visit(node ast.Node) ast.Visitor {
+type FuncInfo struct {
+	HasDefer bool
+	// Nodes are "flattened" into a switch-case statement when we need to be able
+	// to jump into an arbitrary position in the code with a GOTO statement, or
+	// resume a goroutine after a blocking call unblocks.
+	Flattened map[ast.Node]bool
+	// Blocking indicates that either the AST node itself or its may block
```

Comment:
I think this sentence is missing a word?  Doesn't quite make sense.

Reply:
Good catch, fixed.